torch/_meta_registrations.py

```diff
@@ -2508,6 +2508,7 @@ def meta_mkl_linear(input_tensor, packed_weight, orig_weight, bias, batch_size):
     )
 
     @register_meta(torch.ops.onednn.qconv2d_pointwise.default)
+    @register_meta(torch.ops.onednn.qconv_pointwise.default)
     def meta_qconv2d_pointwise(
```

Comment:
change the func name to qconv?

Reply:
It's qconv now. Which func name are you referring to?

Reply:
I mean the name of this meta function `meta_qconv2d_pointwise`

Reply:
Ok. Changed. Thanks.